test/no_volumes_submission.json

```diff
@@ -0,0 +1,108 @@
+{
```

Comment:
(this file is copied from the `model` repo, but it's just the same as the other test submission file, but without volume info)